src/omnicore/rpc.cpp

```diff
@@ -144,6 +144,65 @@ bool BalanceToJSON(const std::string& address, uint32_t property, Object& balanc
     }
 }
 
+// generate a list of seed blocks based on the data in LevelDB
+Value omni_getseedblocks(const Array& params, bool fHelp)
+{
+    if (fHelp || params.size() < 2 || params.size() > 3)
+        throw runtime_error(
+            "omni_getseedblocks startblock endblock\n"
+            "\nReturns a list of blocks containing Omni transactions for use in seed block filtering.\n"
+            "\nWARNING: The Exodus crowdsale is not stored in LevelDB, thus this is currently only safe to use to generate seed blocks after block 255365"
+            "\nArguments:\n"
+            "1. startblock           (number, required) the first block to look for Omni transactions (inclusive)\n"
+            "2. endblock             (number, required) the last block to look for Omni transactions (inclusive)\n"
+            "3. stringify            (bool, optional) whether to output one continuous string of seed blocks for easier inclusion in seedblocks.cpp\n"
+            "\nResult:\n"
+            "[                           (array of JSON objects)\n"
+            "  {\n"
+            "    \"seedblock\" : blockheight,    (number) the blockheight of the seed block\n"
+            "  },\n"
+            "  ...\n"
+            "]\n"
+            "\nExamples:\n"
+            + HelpExampleCli("omni_getseedblocks", "290000 300000")
+            + HelpExampleRpc("omni_getseedblocks", "290000, 300000")
+        );
+
+    int startHeight = params[0].get_int();
+    int endHeight = params[1].get_int();
+    bool overrideOutput = false;
+    if (params.size() > 2) overrideOutput = params[2].get_bool();
+
+    RequireHeightInChain(startHeight);
+    RequireHeightInChain(endHeight);
+
+    std::set<int> setSeedBlocks = p_txlistdb->GetSeedBlocks(startHeight, endHeight);
```

Comment:
Let's do it here, to guard the pointer `p_txlistdb` too. Not sure, if this is the best way, but it's similar to other places and it might come handy in case of a shutdown (where the pointer is cleared).